### ISSUE TITLE
Roster locks in a season

### ISSUE THREAD

**Issue by superjova** [chairleague/hots-rules#4]
Add roster locks to teams so ringers cannot be added.

Lock rosters of teams who are signed up for divisions on week 6.

**Comment by Durgulach** [comment]
I like having a cutoff, akes for a more interesting end of season free from ringers.  2 questions

1. Does on week 6 mean after week 5 is complete or after week 6 is complete (aka will rosters be locked as of start of game 6 or game 7?)

2.  How are we reconciling the option to sub with this rule?  I cant remember if we said 5/5 must be on roster or not, if only 4/5 that would be a loophole to this roster lock.

**Comment by Ronin89** [comment]
I believe we said 5/5, but could use a confirmation.

**Comment by Hammwich** [comment]
Late to the party, but can say from my personal meeting notes that I understood it to be 5/5 players on a teams roster at the end of Week 6. 

In other words, no additional players may be added to a teams roster after week 6, this includes post season play. 

So 6 weeks to find 3 subs, after that 8/8 of your player slots should be inclusive of your starting 5 and any potential subs.

**Comment by Durgulach** [comment]
The latest I have seen has confirmed 4/5 to start a match with no language indicating a change after roster lock.  So I guess you can backdoor a single player after roster lock as a sub. 

Really that probably isnt the worst thing because it is a rec league and nothing would suck worse than having to forfit a playoff match due to irl conflicts resulting in only having 4/5

**Comment by Hammwich** [comment]
> Really that probably isnt the worst thing because it is a rec league and nothing would suck worse than having to forfit a playoff match due to irl conflicts resulting in only having 4/5

I think with 8 roster slots it forces players to commit, if you're allowing subs come playoffs, whats to stop a team from removing their lowest rated player in favor of someone much better, just in time to win the league? Its creates this wild west mentality that many of my teammates oppose. 5 starters and 3 subs should allow for any team to address player absences.

Again consider the HGC, they allow for subs for a set number of matches, no subs are allowed to play during the playoffs. League Structure, and Team need to mean more than they ever have now that players are paying to be a part of the league.

**Comment by Durgulach** [comment]
I'm with you but I'm torn.  I have no problem going no-sub post roster lock, but then again I am on a team with 8 players.  5 man squads may struggle but we would have to see what happens.

Either way if we are going to force 5/5 for playoffs (or post week 5 after lock) then we must must must put that language in the rules ASAP so we avoid any complaints of communication issues that have already been tossed adound a bit this preseason.  Basically right now there is no firm rules and people are registering which makes me uneasy from a customer service standpoint.

**Comment by Hammwich** [comment]
Agreed, we need get the rules in place if they are going to be there at all. @superjova looking for some feedback from you on this one.

**Comment by Ronin89** [comment]
I agree more with Hammwich on this one.  5 weeks after it starts should be
plenty of time to get 3 extra guys in case of a needed sub.

**Comment by Hammwich** [comment]
Trying to bring this issue to the top of the pile, I'm seeing a few teams with entirely new lineups as we enter the playoffs. I don't think this promotes healthy competition and without any restrictions on player additions leaves more room for things like the TigerJK incident to occur. This should be addressed for all future seasons.
